pfsimulator/parflow_lib/metadata.h

```diff
@@ -0,0 +1,66 @@
+#ifndef pf_metadata_h
```

Comment:
Please include GNU license and optional copyright statement if one is needed by your organization.

Reply:
I've pushed a commit with the licenses/copyrights. Sorry it has taken so long.